docs/resources.md

```diff
@@ -618,6 +619,213 @@ the container image
 [gcr.io/cloud-builders//gcs-fetcher](https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/gcs-fetcher)
 does not support configuring secrets.
 
+### Cloud Event Resource
+
+The Cloud Event Resource represents a cloud event that is sent to a target `URI`
```

Comment:
would you mind linking to some docs here about what Cloud Events are in general?

Reply:
yeah, good point, thank you